src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -84,6 +85,13 @@ public ActiveSolutionBoundTracker(IHost host, IActiveSolutionTracker activeSolut
             CurrentConfiguration = configurationProvider.GetConfiguration();
 
             SetBoundSolutionUIContext();
+
+            gitEventsMonitor.HeadChanged += GitEventsMonitor_HeadChanged;
+        }
+
+        private void GitEventsMonitor_HeadChanged(object sender, EventArgs e)
+        {
+            SolutionBindingUpdated?.Invoke(this, EventArgs.Empty);
```

Comment:
This is only used in Taint and suppressions so reusing it makes sense

Reply:
I don't think that necessarily follows, but I think it's ok for now.